src/main/java/org/apache/commons/lang3/time/GmtTimeZone.java

```diff
@@ -31,6 +36,32 @@
     private static final int MINUTES_PER_HOUR = 60;
     private static final int HOURS_PER_DAY = 24;
 
+    private static final TimeZone timezone = new GmtTimeZone(false,0,0);
+    private static final Pattern GMT_PATTERN1 = Pattern.compile("^(?:(?i)GMT)?([+-])?(\\d\\d?)?(:?(\\d\\d?))?$");
+
+    public static TimeZone fetchGmtTimeZone(final String pattern) {
```

Comment:
Other methods in this class follow the get/set pattern. This `fetchGmt...` method is being called from `FastTimeZone#getGmtTimeZone`. So it could probably stay as `getGmt...`, I think.